hydromt_wflow/wflow.py

```diff
@@ -2203,158 +2291,86 @@ def setup_reservoirs(
             in the netcdf file.
         geom_name : str, optional
             Name of the reservoirs geometry in the staticgeoms folder, by default
-            "reservoirs" for reservoirs.geojson.
+            "reservoirs_simple_control" for reservoirs_simple_control.geojson.
         kwargs: optional
             Keyword arguments passed to the method
             hydromt.DataCatalog.get_rasterdataset()
+        """  # noqa: E501
+        # retrieve data for basin
+        self.logger.info("Preparing reservoir with simple control maps.")
+        if "predicate" not in kwargs:
+            kwargs.update(predicate="contains")
```

Comment:
https://docs.python.org/3/library/stdtypes.html#dict.setdefault

```suggestion
        kwargs.setdefault("predicate", "contains")
```